src/main/java/com/mide/gangsaeng/board/BoardCacheRepositoryImpl.java

```diff
@@ -39,6 +39,19 @@ public void write(Board board) {
         }
     }
 
+    @Override
+    public void update(Board board) {
+
+        try {
+            final long boardId = board.getId();
+            redisCommands.del(makeBoardKey(boardId));
+        } catch (RedisException e) {
+            log.error(e.getMessage(), e);
+        }
+
+        this.write(board);
+    }
+
```

Comment:
I think we can just simply overwrite the cache value.

Reply:
According to docs, if key already holds a value, it is overwritten, regardless of its type. 
So i changed.
[https://redis.io/docs/latest/commands/set/](url)